p4runtime_sh/bytes_utils.py

```diff
@@ -105,6 +109,8 @@ def make_canonical_if_option_set(bytes_):
 
 
 def parse_value(value_str, bitwidth, base=0):
+    if bitwidth == 0:
+        return str_to_bytes(value_str)
```

Comment:
In which case is the bitwidth 0 in the P4Info?

Reply:
When creating new types like this.
@p4runtime_translation("", string)
type bit<VRF_BITWIDTH> vrf_id_t;

The generated P4info does not have the bitwidth specified.
  new_types {
    key: "vrf_id_t"
    value {
      translated_type {
        sdn_string {
        }
      }
    }
  }